server/plugin/template.go

```diff
@@ -351,6 +351,13 @@ Assignees: {{range $i, $el := .Assignees -}} {{- if $i}}, {{end}}{{template "use
 		"  * `/github mute add [username]` - add a GitHub user to your muted list\n" +
 		"  * `/github mute delete [username]` - remove a GitHub user from your muted list\n" +
 		"  * `/github mute delete-all` - unmute all GitHub users\n"))
+
+	template.Must(masterTemplate.New("newRepoStar").Funcs(funcMap).Parse(`
+{{template "repo" .GetRepo}}
+{{- if eq .GetAction "created" }} Starred
+{{- else }} Unstarred
+{{- end }} by {{template "user" .GetSender}}
+Now, **{{.GetRepo.GetStargazersCount}}** Stars.`))
```

Comment:
Maybe the wording can be a bit more natural. 
```suggestion
It now has **{{.GetRepo.GetStargazersCount}}** Stars.`))
```